src/main/java/newbank/client/ExampleClient.java

```diff
@@ -111,6 +112,11 @@ private void displayWelcomingBanner() {
     display.writeLine(message);
   }
 
+  private void welcomeMenu() {
+    String menu = "Please login to your account ('LOGIN username password') or register ('REGISTER username password')";
```

Comment:
Please catch up with main as @claudist has already checked that in.

Regards,
Jules

Reply:
Resolved.